apps/dashboard/app/models/posix_file.rb

```diff
@@ -85,6 +85,10 @@ def directory?
   def ls
     path.each_child.map do |child_path|
       PosixFile.stat(child_path)
+    end.select do |stats|
+      ascii_only = stats[:name].to_s.ascii_only?
```

Comment:
`ascii_only?` puts a lot of UTF-8 characters in the wrong bin, I'm afraid. I tried this code on our sandbox instance, and for example these strings get ignored in error (i.e., they should be allowed):
```
App 22009 output: [2023-03-02 12:06:19 +0200 ]  WARN "Not showing file 'これは動作しません' because it is not a UTF-8 filename."
App 22009 output: [2023-03-02 12:06:19 +0200 ]  WARN "Not showing file 'тест' because it is not a UTF-8 filename."
```